api/hearings/data/reference.mock.data.ts

```diff
@@ -266,6 +394,10 @@ export const ADDITIONAL_HEARING_FACILITIES: RefDataByServiceModel = {
   serviceID: 'SSCS',
   values: ADDITIONAL_HEARING_FACILITIES_REF,
 };
+export const LIST_OFF_VALUES: RefDataByServiceModel = {
```

Comment:
As this is a generic API LoV(List Of Values), this LIST_OFF_VALUES is too generic here, it should be something specific to SSCS_OTHER_PANEL_ROLES